attesters/hydra-s1-simple/index.ts

```diff
@@ -1,11 +1,19 @@
 import { AttestationsCollection } from "../../src/topics/attestations-collection/attestations-collection";
-import { Badge } from "../../src/topics/badge";
 import { Attester } from "../../src/topics/attester";
+import { AttesterNetwork } from "../../src/topics/attester/types";
+import { Badge } from "../../src/topics/badge";
 import { Group } from "../../src/topics/group";
 
-export const hydraS1SimpleAttester = async () =>
+export default async () =>
   new Attester({
-    collectionIdFirst: 10000001,
+    name: "hydra-s1-simple",
+    configurations: {
```

Comment:
rename `configurations` to `networkConfiguration` ?
Or have something like:
```
configuration: {
  network: {}
}
```
@leosayous21